packages/easy-testing/src/DataProvider/StaticFixtureFinder.php

```diff
@@ -8,13 +8,21 @@
 use Symfony\Component\Finder\Finder;
 use Symfony\Component\Finder\SplFileInfo;
 use Symplify\SmartFileSystem\SmartFileInfo;
+use Symplify\SymplifyKernel\Exception\ShouldNotHappenException;
 
 final class StaticFixtureFinder
 {
     public static function yieldDirectory(string $directory, string $suffix = '*.php.inc'): Iterator
     {
         $fileInfos = self::findFilesInDirectory($directory, $suffix);
+        foreach ($fileInfos as $fileInfo) {
+            yield [new SmartFileInfo($fileInfo->getRealPath())];
+        }
+    }
 
+    public static function yieldDirectoryExclusively(string $directory, string $suffix = '*.php.inc'): Iterator
```

Comment:
:+1: 

This method will need a test case in a new `StaticFixtureFinderTest` class

Reply:
implemented.